client/js/api-reference/messages.mdx

```diff
@@ -1,45 +1,288 @@
 ---
-title: "Messages"
+title: "Custom Messaging"
```

Comment:
See earlier note about the need for "custom"

Reply:
same response. I'm happy either way, but let me know your thoughts after reading why I used "Custom messaging" here.